github-policies.md

```diff
@@ -35,10 +35,21 @@ In this GitHub organization, you'll find:
   - [the Swift Code of Conduct](https://www.swift.org/code-of-conduct/)
   - [the Swift Security process](https://www.swift.org/support/security.html) unless otherwise noted in the projects approved SECURITY.md at the root.
 - All projects agree to open and active development.
-  - Follows the Swift Toolchain Release Process; if not, there is a documented process document.
-      - For projects that are packages:
-        - There should be a statement in the README on how those packages handle versioning.
-        - The package supports the latest official Swift release and one prior.
+- Projects are either released as part of the Swift Toolchain or as Swift packages.
+- Projects released through the Swift Toolchain follow the Swift Toolchain Release Process.
+- For projects released as Swift packages:
+  - Package versions **must** follow [Semantic Versioning](https://semver.org).
+  - Packages **should** support the **latest patch releases** of the **last three minor** official Swift releases.
+    - For example, if the currently latest Swift release is `6.2.1` then a package would be expected to support `6.2.1`, `6.1.3` and `6.0.3`.
+    - As an exception, new packages are free to start with only supporting the latest official Swift release.
+    - The expected Swift support should be tested on every pull request using the continuous integration provided by [swiftlang/github-workflows](https://github.com/swiftlang/github-workflows).
```

Comment:
Would be great if github-workflows contained a single meta-workflow that a maintainer can add to their package's workflow, which runs tests on all the supported Swift version * Swift supported platform matrix, with the ability to opt-out of specific platforms/swift versions. Otherwise whenever a new platform is added, all packages need to be updated, which might lead to inconsistent platform support.